### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by martinmckenna**
## Description

Update release instructions

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -74,16 +74,22 @@ When a new version of Cloud Manager is released, it must be accompanied by an up
...
 4. Review the Changelog and update manually if necessary
-5. Stage and commit the version bump and changelog addition with the commit message, which should be as simple as: `vX.X.X`
-6. Push up the changes to the `testing` branch with `git push origin testing`
-7. Finally, follow the merge flow. Merge from `testing` to `staging` and finally to `master`
+5. Once your Changelog has been approved by the team, stage the changes locally and run `yarn version --new-version X.X.X` (replace the X's with the appropriate version number)
+    * this will apply the Git tags and update the version number in the `package.json`
+    * This will also automatically commit the changes with the commit message `vX.X.X`
+6. Push the changes from your local `testing` branch to the upstream
+    * You may need to add the `--no-verify` flag, as the `testing` branch isn't prefixed with `M3`
+7. At last, follow the merge flow. Merge from `testing` to `staging` and finally to `master`
```
Comment (Jskobos) [inline CONTRIBUTING.md]: I get this but for future devs might help to be more explicit (give command examples etc)

Comment (martinmckenna) [reply CONTRIBUTING.md]: Fair and will update

Comment (martinmckenna) [reply CONTRIBUTING.md]: updated

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -74,16 +74,36 @@ When a new version of Cloud Manager is released, it must be accompanied by an up
...
+    * This includes getting rid of any references to PR numbers, JIRA ticket numbers or grammar and spelling mistakes
+    * You should also ensure that everything in the Chaneglog is user-facing. Removing anything that users won't directly be interacting with
+5. Once your Changelog has been approved by the team, run `git add . && yarn version --new-version X.X.X` (replace the X's with the appropriate version number)
+    * This will apply the Git tags and update the version number in the `package.json`
+    * This will also automatically commit the changes with the commit message `vX.X.X`
+6. Push the changes from your local `testing` branch to the upstream with `git push origin testing`
+    * You may need to add the `--no-verify` flag, as the `testing` branch isn't prefixed with `M3`
+7. At last, follow the merge flow as normal
+    * Merge `testing` into `staging` with `git checkout staging && git pull && git merge testing && git push origin staging`
+    * Merge `staging` into `master` with `git checkout master && git pull && git merge staging && git push origin master`
```
Comment (johnwcallahan) [inline CONTRIBUTING.md]: I'd make this it's own step/number, and make it clear that this is what actually releases it to production.

**Pull request merged**